spec/integration/devtools/rake/flay/verify_spec.rb

```diff
@@ -0,0 +1,171 @@
+require 'spec_helper'
+require 'tempfile'
+require 'tmpdir'
+require 'flay'
+
+# rubocop:disable ModuleLength
+module Devtools
```

Comment:
I think that this module re-opening should not be done, and the describe should be `describe Devtools::Rake::Flay`. This is not only shorter and "saves an indentation level". But also prevents accidentally mutating the "production namespace" as it is not opened.